src/Components/Publishing/RelatedArticles/RelatedArticlesCanvas.tsx

```diff
@@ -47,7 +48,7 @@ export const RelatedArticlesCanvas: React.SFC<RelatedArticlesCanvasProps> = prop
             })}
           </ArticlesWrapper>
         </RelatedArticlesContainer>
-      </div>
+      </StandardLayoutParent>
```

Comment:
@kanaabe - i wrapped a few problematic areas in this new wrapper. At some point we could probably refactor the layout so that this is built into the overall container instead of needing to be applied to independent elements.

Reply:
Because features need to be full screen width I think we will still need to wrap the display/related units to give them a margin.  Maybe we could use a more generic name for the standard element padding?

Reply:
Will revisit in another PR